packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -11,6 +11,8 @@ export const verticalPillsBottomMargin = pxToRem(5);
 export const horizontalPillsRightMargin = pxToRem(8);
 export const verticalPointingBottomMargin = pxToRem(12);
 
+export const menuItemIconClassName = 'ui-menu__itemicon';
```

Comment:
Please import this from `packages/fluentui/react-northstar/src/components/Menu/MenuItemIcon.tsx`:

```ts
import { menuItemIconClassName } from '../../../../components/Menu/MenuItemIcon';
```